sys/net/crosslayer/ng_netapi/ng_netconf_debug.c

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ * @ingroup     net_ng_netapi
+ * @file
+ * @brief       This file contains functionality to map netconf option
+ *              numbers to strings
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ * @}
+ */
+#include "net/ng_netconf.h"
+
```

Comment:
C++ guards are missing, which is very important here due to not using C++ syntax below.